terra/views.py

```diff
@@ -121,6 +122,51 @@ def get_context_data(self, *args, **kwargs):
         return context
 
 
+class FundExportView(FundDetailView):
+    def render_to_response(self, context, **response_kwargs):
+        fund = context.get("fund")
+        fy = context.get("fiscalyear", "").replace(" ", "")
+        totals = context.get("totals")
+        response = HttpResponse(content_type="text/csv")
+        response["Content-Disposition"] = f'attachment; filename={fund}_FY{fy}.csv"'
```

Comment:
@joshuago78 On my Windows system, downloading a file in Firefox/Chrome/Edge all result in a `.csv_` file - there's a trailing underscore, like `605000-LD-18084_FY2019-2020.csv_`.  This prevents the file from automatically opening in Excel or other applications which expect `.csv` with no underscore.

I believe this is caused by the final double-quote in this line, which is not valid in filenames on Windows, so apparently the browser converts it to an underscore.

Reply:
Good catch. It's missing the opening quote. I just pushed a fix